lua/controller.lua

```diff
@@ -0,0 +1,377 @@
+--[[
+Minetest Mod Storage Drawers - A Mod adding storage drawers
+
+Copyright (C) 2018 isaiah658
+Copyright (C) 2018 LNJ <git@lnj.li>

```

Comment:
I actually haven't worked on this file in 2018, so please either use `2017-2018` or `2017`.